extensions/amp-base-carousel/0.1/test-e2e/test-default-attributes.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.endtoend(
+  'AMP carousel test default attributes',
+  {
+    testUrl:
+      'http://localhost:8000/test/manual/amp-base-carousel/default-attributes.amp.html',
+    environments: ['single'],
```

Comment:
Can this be enabled for `viewer-demo` as well? I know `shadow-demo` tends to be skipped for these

Reply:
Yea `viewer-demo` dies:
```
     Error: Waiting for element to be located #viewer[data-loaded]
```

I'll bring it up with the infra team. For now, I'll add a TODO.